app/models/compute_resources/foreman/model/ovirt.rb

```diff
@@ -695,5 +696,27 @@ def set_vm_interfaces_attributes(vm, vm_attrs)
       end
       vm_attrs
     end
+
+    def set_vm_volumes_attributes(vm, vm_attrs)
+      template = template(vm_attrs[:template]) if vm_attrs[:template]
+      template_volumes_name_id = {}
+      template&.volumes&.each { |volume| template_volumes_name_id[volume.name] = volume.id }
+      volumes = vm.volumes || []
+      vm_attrs[:volumes_attributes] = volumes.each_with_index.each_with_object({}) do |(volume, index), hsh|
+        id = nil
+        id = template_volumes_name_id[volume.name] if template_volumes_name_id.key? volume.name
+        volume_attrs = {
+          size_gb: (volume.size.to_i / DISK_SIZE_TO_GB),
```

Comment:
Rails already has a helper for this:
```suggestion
          size_gb: (volume.size.to_i / 1.gigabyte),
```